packages/jest-mock/src/index.js

```diff
@@ -430,9 +430,7 @@ module.exports = {
    * @see README.md
    * @param component The component for which to retrieve metadata.
    */
-  getMetadata(component) {
-    return getMetadata(component);
-  },
+  getMetadata,
```

Comment:
This shouldn't really be part of this diff but since it's a straightforward and minor change, I'm fine with it :)